### ISSUE TITLE
WoW TBC - Outland Content

### ISSUE THREAD

**Issue by Wyr3d** [smaitch/Grail#282]
No Outland content displays in the current version, those sections are just missing. While I realise that Classic isn't the same as Retail, can't most of the information be taken from there?

**Comment by smaitch** [comment]
The choice was made to specifically not take retail information to populate classic.  I am hoping people will start sending in their account-wide Grail.lua saved variables file to allow me to populate those zones faster.

**Comment by Wyr3d** [comment]
Fair enough, but this might be of use to you. I believe it is written for Classic TBC:

https://www.warcrafttavern.com/tbc/guides/horde-leveling-guide/

Has both factions, but Horde is best faction :P

**Comment by Wyr3d** [comment]
So then, are still waiting on Grail.lua files? I can send mine if you like, I've done most of the leveling quests, so it's pretty much just end zone, and end game stuff I still have to do. Where do I send it?

**Comment by yoshimo** [comment]
you can attach it here to the bug
